cypresse2e/cypress/elements/pages/aboutUs.js

```diff
@@ -13,12 +13,16 @@ class AboutUs {
   joinStudentProgramLink = "https://apply.workable.com/fightpandemics/j/58B157AAB2/";
   getToHelpBoardButton = "#AU_VHB";
   goToHelpBoardLink = "/feed";
-  lifetimeSupportersContainerXpath = "//*[@id=\"root\"]/div/div/div[3]/main/div/div[8]/div[1]";
-  currentSupportersContainerXpath = "//*[@id=\"root\"]/div/div/div[3]/main/div/div[8]/div[2]";
-  pastSupportersContainerXpath = "//*[@id=\"root\"]/div/div/div[3]/main/div/div[8]/div[3]";
+  getToPressKitButton = "#root > div > div > div.am-drawer-content > main > div > div.sc-jOFreG.dCiTET > div > a";
```

Comment:
Those classNames (exp: `.sc-jOFreG.dCiTET`) are randomly generated for each build and are not consistent.